php/ext/google/protobuf/package.xml

```diff
@@ -10,15 +10,15 @@
   <email>protobuf-opensource@google.com</email>
   <active>yes</active>
  </lead>
- <date>2021-08-27</date>
- <time>14:37:43</time>
+ <date>2021-09-13</date>
+ <time>11:30:58</time>
  <version>
-  <release>3.18.0RC2</release>
+  <release>3.18.0</release>
   <api>3.18.0</api>
  </version>
  <stability>
-  <release>beta</release>
-  <api>beta</api>
+  <release>stable</release>
+  <api>stable</api>
  </stability>
  <license uri="https://opensource.org/licenses/BSD-3-Clause">3-Clause BSD License</license>
  <notes>
```

Comment:
This PHP notes field currently has to be updated by hand.

Reply:
updated